unittest/test_solver_statistics.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
I wonder if this is an issue when running with robostack (i.e., in a conda environment).
For instance, the default python version in my M1 OSX is v3.10. However, ros-noetic in robostack needs to be created with a python v3.9 environment.

Reply:
It shouldn't be an issue since those environments should remap the environment python to the locally used one - but please let me know should it cause an issue with `catkin run_tests crocoddyl_msgs`